### PR TITLE
Component/delete button

### LINKED ISSUE #7: DeleteButton component
Component for #2

### PR DESCRIPTION
**Pull request by PReichetanz**
Add DeleteButton as controlled component
- Controlled parts: content and deleteFunction
- Add style to button
- Add two stories ("X" and "–")

Bildschirmfoto 2021-11-24 um 09 42 42
Bildschirmfoto 2021-11-24 um 09 43 07

Closes #7

### CODE REVIEW

File: src/app/components/DeleteButton/DeleteButton.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+
+type DeleteButtonProps = {
+  handleDelete: () => void;
+  content: string;
```
Comment (marwinburesch) [inline src/app/components/DeleteButton/DeleteButton.tsx]: Maybe you want to use `children` here, that way you could add an SVG to display on the button in the future

File: src/app/components/DeleteButton/DeleteButton.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  padding: 0.625rem;
+  position: absolute;
+  right: 0.25rem;
+  top: 0.25rem;
+  &:hover {
+    background: var(--color-tertiary-hover);
+  }
+`;
+
+const Delete = styled.button`
```
Comment (marwinburesch) [inline src/app/components/DeleteButton/DeleteButton.tsx]: Not the best name for a styled button in my opinion

**Review by marwinburesch — COMMENTED** [review]
Nice!
I have a little food for thought here though: If you'll ever need a similar looking button, you could turn this into a `RoundActionButton`, or in other words make this a tad more generic and you'll have a versatile round button :)

File: src/app/components/DeleteButton/DeleteButton.tsx (hunk 1 above)
Comment (PReichetanz) [reply src/app/components/DeleteButton/DeleteButton.tsx]: Very good idea, thank you!

File: src/app/components/DeleteButton/DeleteButton.tsx (hunk 2 above)
Comment (PReichetanz) [reply src/app/components/DeleteButton/DeleteButton.tsx]: True, I changed it.

File: src/app/components/DeleteButton/DeleteButton.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: var(--color-tertiary);
+  border-radius: 50%;
+  border: 1px solid var(--color-stroke);
+  padding: 0.625rem;
+  position: absolute;
```
Comment (lajanzen) [inline src/app/components/DeleteButton/DeleteButton.tsx]: Not sure if I would make the whole component absolute. I'd rather leave this out here and when you integrate the component on the page, set the value there.

**Review by lajanzen — COMMENTED** [review]
I think that the way you constructed the button with a container, the only "clickable" part of the button is actually the "X" or "-". 
In case you want the whole container to be clickable, I would suggest to remove the container and just style the whole button as such, so the whole component is clickable.

**Pull request merged**